plugins.js

```diff
@@ -212,18 +210,18 @@ Plugin.prototype._get_code = function (pp) {
             // escape the c:\path\back\slashes else they disappear
             packageDir = packageDir.replace(/\\/g, '\\\\');
         }
-        return 'var _p = require("' + packageDir + '"); for (var k in _p) { exports[k] = _p[k] }';
+        return `var _p = require(`${packageDir}`); for (var k in _p) { exports[k] = _p[k] }`;
```

Comment:
This breaks loading plugins because it changes `require('foo')` to `require(foo)`. We still need the quotes.